script/scaffold/__main__.py

```diff
@@ -60,28 +60,24 @@ def main() -> int:
 
             generate.generate(template, info)
 
-    hassfest_args = [
-        "python",
-        "-m",
-        "script.hassfest",
-    ]
-
     # If we wanted a new integration, we've already done our work.
     if args.template != "integration":
         generate.generate(args.template, info)
-    else:
-        hassfest_args.extend(
-            [
-                "--integration-path",
-                info.integration_dir,
-                "--skip-plugins",
-                "quality_scale",  # Skip quality scale as it will fail for newly generated integrations.
-            ]
-        )
 
     # Always output sub commands as the output will contain useful information if a command fails.
     print("Running hassfest to pick up new information.")
-    subprocess.run(hassfest_args, check=True)
+    subprocess.run(
```

Comment:
can we catch the exception so we don't print a stack trace if it were to go wrong. Maybe even print an error that this is a bug and should be reported?

Reply:
Have refactored to add error checking and reporting so the output is more understandable for users.